src/components/UserInfo.js

```diff
@@ -10,9 +11,17 @@ export default class UserInfo {
         name: this.name.textContent,
       };
     };
-  
-    setUserInfo(name, profession) {
-      this.name.textContent = name;
-      this.profession.textContent = profession;
+
+    setUserInfo({name, about, avatar, _id}) {
```

Comment:
Good job. The class has been nicely implemented. Most students have problems here.